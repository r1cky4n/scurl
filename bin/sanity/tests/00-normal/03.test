https://dh2048.badssl.com/
