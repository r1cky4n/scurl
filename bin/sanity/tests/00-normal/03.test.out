<!DOCTYPE html>
<html>
<head>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="/icons/favicon-green.ico"/>
  <link rel="apple-touch-icon" href="/icons/icon-green.png"/>
  <title>dh-2048.badssl.com</title>
  <link rel="stylesheet" href="/style.css">
  <style>body { background: green; }</style>
</head>
<body>
<div id="content">
  <h1>
    dh-2048.badssl.com
  </h1>
</div>

<div id="footer">
  This site uses an ephemeral Diffie-Hellman key exchange<br>over a 2048-bit group.
</div>

</body>
</html>
